temporal-normalizer/TimeML.dtd

```diff
@@ -1,3 +1,12 @@
+<!--
```

Comment:
Do you want to drop this license header? You can add exception to `dtd` files: 
https://github.com/CogComp/cogcomp-nlp/blob/master/pom.xml#L156-L167